tests/unit/Utils/DPOPTest.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Pdsinterop\Solid\Auth\Utils;
+
+use PHPUnit\Framework\TestCase;
+use Lcobucci\JWT\Validation\RequiredConstraintsViolated;
+
+/**
+ * @covers DPop
```

Comment:
This won't work, as PHPUnit requires a FQN: `\Pdsinterop\Solid\Auth\Utils\DPop`.

Also... For the class-level `@covers` you might want to opt for:

```
/*
 * @coversDefaultClass \Pdsinterop\Solid\Auth\Utils\DPop
 * @covers ::__construct
 * @covers ::<!public>
 */
 ```
 
 So you do not have to repeat the FQN for method-level tests:
 ```
/**
 * @covers ::validateDpop
 */
```

Also, as the `Base64Url` class is used, a `@uses` annotation will also have to be added (either here or at method level).

Reply:
done